src/pages/Main/components/BoxOfficeCarousel.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import MovieCarousel from './MovieCarousel';
+import useFetch from '../../../hooks/useFetch';
+
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const BoxOfficeCarousel = () => {
+  const [sources] = useFetch('data/movieCarouselData.json');
```

Comment:
위 리뷰대로 구현하신다면 다음과 같이 받아오실 수 있습니다.
```suggestion
  const { data, isLoading, error } = useFetch('data/movieCarouselData.json');
```

Reply:
이부분에 대해 내용이 이해가 되지 않아 이따 직접 질문하겠습니다!